subliminal/core.py

```diff
@@ -703,3 +713,12 @@ def save_subtitles(video, subtitles, single=False, directory=None, encoding=None
             break
 
     return saved_subtitles
+
+
+def raise_for_status(r):
+    if r.status_code == 304:
+        raise TooManyRequests()
+    elif r.status_code == 503:
+        raise ServiceUnavailable
+    else:
+        r.raise_for_status()
```

Comment:
OpenSubtitles does that because there is no `raise_for_status` available. I agree that ServiceUnavailable shouldn't be related to OpenSubtitles though but maybe a broader exception class within whole subliminal.